src/api/api.ts

```diff
@@ -1,26 +1,83 @@
 import axios, { AxiosRequestConfig } from "axios";
+import moment from "moment";
 
 import { ENV } from "@app/constants/env";
 import { AuthEndpointsEnum, getTokens } from "@app/features/auth/auth";
 
+const isRefreshing = false;
+
 /**
  * All the endpoint that do not require an access token
  */
 const anonymousEndpoints = [AuthEndpointsEnum.LOGIN.toString()];
 
+/**
```

Comment:
This file is prepared for a refreshToken functionality. The functions and wrappers are there, but theres no actual refresh. 
We can leave the code there for the next project?